CMakeLists.txt

```diff
@@ -33,21 +32,21 @@ add_library(pointcloud_filters src/pointcloud_filters.cpp)
 target_link_libraries(pointcloud_filters ${catkin_LIBRARIES})
 
 add_library(laser_scan_filters src/laser_scan_filters.cpp src/median_filter.cpp src/array_filter.cpp src/box_filter.cpp)
-target_link_libraries(laser_scan_filters ${catkin_LIBRARIES} ${Boost_LIBRARIES} ${Eigen_LIBRARIES})
+target_link_libraries(laser_scan_filters ${catkin_LIBRARIES})
```

Comment:
ok。sure。but i tested my changes, and test result shows my modification is right and can run correctly.
I'll change back to your style later.